doc/sphinx-guides/source/user/account.rst

```diff
@@ -134,10 +135,10 @@ Convert your Dataverse account away from ORCID for log in
 
 If you don't want to log in to Dataverse using ORCID any more, you will want to convert your Dataverse account to the Dataverse Username/Email log in option. To do this, you will need to contact support for the Dataverse installation you are using. On your account page, there is a link that will open a popup form to contact support for assistance.
 
-GitHub and Google Log In
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Microsoft Azure AD, GitHub, and Google Log In
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can also convert your Dataverse account to use authentication provided by GitHub or Google. These options may be found in the "Other options" section of the log in page, and function similarly to how ORCID is outlined above. If you would like to convert your account away from using one of these services for log in, then you can follow the same steps as listed above for converting away from the ORCID log in.
+You can also convert your Dataverse account to use authentication provided by GitHub, Microsoft, or Google. These options may be found in the "Other options" section of the log in page, and function similarly to how ORCID is outlined above. If you would like to convert your account away from using one of these services for log in, then you can follow the same steps as listed above for converting away from the ORCID log in.
```

Comment:
@djbrooke now there is a mix of "Microsoft" and "Microsoft Azure AD". Is that helpfull for a user? Admins need precise technical things, but users will have "Microsoft" on that login button. Should we change that button, too? I'm not sure a user will know what "Microsoft Azure AD" is and what they can do with it.

Reply:
We will plan to rename the button, this is a big part of what I was punting. :) Not sure if Microsoft Azure AD button will make sense to users, either.

Reply:
Note to self: Maybe we'll just keep the button as Microsoft. Seems pretty common. 

<img width="534" alt="powerschool" src="https://user-images.githubusercontent.com/7773392/67514069-18875780-f66a-11e9-89f3-a2a6cec23f86.png">
<img width="485" alt="edx" src="https://user-images.githubusercontent.com/7773392/67514070-18875780-f66a-11e9-97e1-1054a2ac5bc2.png">
<img width="434" alt="wunderlist" src="https://user-images.githubusercontent.com/7773392/67514071-191fee00-f66a-11e9-965c-06eed005e9ae.png">
<img width="414" alt="doodle" src="https://user-images.githubusercontent.com/7773392/67514073-191fee00-f66a-11e9-935e-1ddbde1315ca.png">